pkg/scheduler/apis/config/v1beta2/default_plugins.go

```diff
@@ -107,34 +107,34 @@ func getDefaultPlugins() *v1beta2.Plugins {
 			},
 		},
 	}
-	applyFeatureGates(plugins)
+	applyFeatureGates(logger, plugins)
 
 	return plugins
 }
 
-func applyFeatureGates(config *v1beta2.Plugins) {
+func applyFeatureGates(logger klog.Logger, config *v1beta2.Plugins) {
```

Comment:
Got this from @pohly initial work. @pohly is the idea here standardization?

Reply:
It probably was used at some point. If that is not the case anymore, then there's no need to add it now because this is an internal function that can still be changed later if (and when) needed.

Reply:
v1beta2 and v1beta3 packages are already marked as deprecated. No need to add now.